hrm-domain/hrm-core/permissions/index.ts

```diff
@@ -15,16 +15,16 @@
  */
 
 import { SafeRouterRequest } from './safe-router';
+import { rulesMap } from './rulesMap';
```

Comment:
Not saying for this PR, but this is exactly the kind of code I'm talking about in the earlier comment when I say shared utility code should be moved out of hrm-core. A `/hrm-domain/packages/hrm-permissions` package should definitely be created at some point (but again, not in this PR)

Reply:
I agree, however the only change added here was the maxPermissions object for internal usage, everything else was already in. I would love to move the permissions into it's own package. For that we need to get in https://github.com/techmatters/hrm/pull/648, in order for the types around PostSurveys and Profiles to also be part of the `types` module, then we can use them here. The other way around does not works, unfortunately.

Mind giving a review to the mentioned PR when you have some spare time? @stephenhand